### DISCUSSION TITLE
Visual mode h,j,k,l -> h,a,e,i

### DISCUSSION THREAD

**Discussion by azizsayfiddin** [philc/vimium#4589] [General]
I use an alternative keyboard layout, and it has different home rows. How can I change h, j, k, l to h, a, e, i for visual mode too?

**Comment by RafayAhmad7548** [comment]
i also want to this, i use ijkl instead of hjkl, i remapped for naormal mode but how do i do it for visual mode

**Comment by Katsuyamaki** [comment]
same, non qwerty keyboard here. Would need to be able to manually remap the keys to make visual mode usable.
